nautobot/docs/additional-features/jobs.md

```diff
@@ -553,57 +553,27 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
 
 Jobs are Python code and can be tested as such, usually via [Django unit-test features](https://docs.djangoproject.com/en/stable/topics/testing/). That said, there are a few useful tricks specific to testing Jobs.
 
-While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. The simplest way to do so is via calling the `nautobot.extras.jobs.run_job()` method, as this is the same function used to execute a Job via Nautobot's Celery worker process. As part of your test, you can call `run_job` directly rather than enqueuing it for the worker to handle; this is simpler and generally quite sufficient for testing.
+While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. Nautobot 1.3 introduced a few enhancements to make this simpler to do, but it's also quite possible to test under Nautobot 1.2 with a bit more effort.
 
-Because of the way `run_job()` works, which is somewhat complex behind the scenes, there are a few things you'll need to do in your test code:
+### Nautobot 1.3 and later
```

Comment:
TODO: This should provide a note that this functionality does not exist in early versions of 1.3. Whether we update it here or leave it as a footnote to do before shipping the next release.